src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputFormattedIncorrectJavadocParagraph.java

```diff
@@ -3,7 +3,7 @@
 /**
  * Some Javadoc.
  *
- * <p>/^ WARN/ Some Javadoc.
+ * <p>Some Javadoc.
```

Comment:
Please change `Some Javadoc.`  to ` some paragraph`.
And summary of javadoc to "Some summary.`.
To make it clear what is what in javadoc content.

To hint user that it is paragraph text.
Please apply to all.

Reply:
done